### ISSUE TITLE
Clarify Gini coefficient

### ISSUE THREAD

**Issue by zhxie** [cesaregarza/SplatTop#16]
The Gini coefficient is 0.6132, indicating a slight imbalance in analytics. But in my education, a coefficient of 0.6 is generally considered relatively high. I also asked ChatGPT and found that a Gini coefficient below 0.3 is indicative of low inequality, while above 0.4 suggests high inequality.

**Comment by cesaregarza** [comment]
You're not wrong, a gini coefficient of 0.61 is definitely extremely imbalanced and in economics would be considered among the worst in the world for income inequality.

I made the decision to slightly bias the interpretation of the gini coefficient on the website to reduce the amount of panic players might exhibit. I don't really have a justification for this, and I might change the values to be more reflective of reality.

**Comment by zhxie** [comment]
Okay, I also calculated the Gini index of Splatoon 2 X rankings after the final update before the release of the 3, and fount that the value is 0.6775, even larger than the 3.
In my mind, weapons in the 2 are more balanced than the 3, so maybe the Gini index cannot prove something that I thought.

**Comment by cesaregarza** [comment]
I wonder how much it’s being affected by the zero usage weapons. I had considered calculating gini throughout the history of the game before, I’ll probably move that up. I’ll also ask top players what they felt the most balanced meta was and calibrate the interpretation based off of that.

**Comment by zhxie** [comment]
Yes, I also did a further look into that. I added weapons with zero usage and merged Hero/Octa Replicas with the original ones. The calculated Gini index is larger than 0.7.
